### ISSUE TITLE
FontAwesome's version update? 

### ISSUE THREAD

**Issue by ivnvitx** [lxndrblz/anatole#80]
I would like (need) to use Mastodon's icon, but it seems that in `head.html` the FontAwesome's version is simply not there. [Here's the link.](https://fontawesome.com/icons/mastodon). 

From [here](https://github.com/FortAwesome/Font-Awesome/milestone/33?closed=1) you can tell it was added in 5.0.11 version. 

I have tried to manually change the FontAwesome version in the link but it seems to be a harder process than just changing the version number in the URL. 

Could you help out on this one?

**Comment by lxndrblz** [comment]
@ivnvitx The FontAwesome version is controlled through the link in the head.html.

```
    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
          integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```

You would need to update this line with the [current Version of the CDN](https://cdnjs.com/libraries/font-awesome) and the correct hash.

Feel free to submit a PR after testing it, so we can roll it out to all users.

**Comment by ivangjxyz** [comment]
I just released the pull request to solve this issue: #81

**Comment by lxndrblz** [comment]
@ivangjxyz I'll have a look. Thanks guys!
